source/uwp/Renderer/lib/WholeItemsPanel.h

```diff
@@ -5,12 +5,11 @@
 
 namespace AdaptiveCards { namespace Rendering { namespace Uwp
 {
-
     class WholeItemsPanel : public Microsoft::WRL::RuntimeClass<ABI::AdaptiveCards::Rendering::Uwp::IWholeItemsPanel,
         ABI::Windows::UI::Xaml::IFrameworkElementOverrides,
         Microsoft::WRL::ComposableBase<ABI::Windows::UI::Xaml::Controls::IPanelFactory>>
     {
-        InspectableClass(L"AdaptiveCards.Uwp.WholeItemsPanel", BaseTrust)
+        InspectableClass(L"AdaptiveCards.Rendering.Uwp.WholeItemsPanel", BaseTrust)
```

Comment:
>AdaptiveCards.Rendering.Uwp.WholeItemsPanel [](start = 27, length = 43)

It seems this was the root cause of the issue. This resulted in the C# projection not being generated correctly, which didn't allow the test app to gather all the xaml image elements inside the panel. That resulted in not waiting for the images to render before creating the image for the test comparison.

Reply:
This absolutely didn't take me a whole day to figure out. Nope. Not at all.

---
In reply to: [154244385](https://github.com/Microsoft/AdaptiveCards/pull/1013#discussion_r154244385) [](ancestors = 154244385)